aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -18,6 +18,22 @@ Tensor quantize_per_tensor_cpu(
   return quantizer->quantize(self);
 }
 
+std::vector<Tensor> quantize_per_tensor_list_cpu(
```

Comment:
@jerryzh168 just confirming that this is the right place to define it? I seem to be hitting linker errors.

Reply:
I think so